mev-relay-rs/src/relay.rs

```diff
@@ -525,10 +550,12 @@ impl BlindedBlockProvider for Relay {
             let local_header = auction_context.signed_builder_bid.message.header();
             if let Err(err) = validate_header_equality(local_header, execution_payload_header) {
                 warn!(%err, %auction_request, "invalid incoming signed blinded beacon block");
-                return Err(RelayError::InvalidSignedBlindedBeaconBlock.into())
+                return Err(RelayError::InvalidSignedBlindedBeaconBlock.into());
```

Comment:
did you run rustfmt via `nightly`? don't think these semis should be in here

Reply:
I'm using `nightly`. For some reason when running `just run-ci` i get an error, requiring a nix fix

Reply:
what error?

Reply:
`cargo +nightly clippy --all-targets --all-features --all
   Compiling c-kzg v0.4.0
error: failed to run custom build command for `c-kzg v0.4.0`

Caused by:
  process didn't exit successfully: `/home/echo/Desktop/Ethereum/mev-rs/target/debug/build/c-kzg-bfe1b3ac0bf2ec00/build-script-build` (exit status: 1)
  --- stderr
  /home/echo/Desktop/Ethereum/mev-rs/target/debug/build/c-kzg-bfe1b3ac0bf2ec00/build-script-build: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.38' not found (required by /home/echo/Desktop/Ethereum/mev-rs/target/debug/build/c-kzg-bfe1b3ac0bf2ec00/build-script-build)
warning: build failed, waiting for other jobs to finish...
error: Recipe `lint` failed on line 15 with exit code 101
`

Reply:
looks like you are missing a dependency for the `c-kzg` crate -- I'd check that repo or just try getting the version of libc it calls out

Reply:
Pop OS doesn't support a newer version of Ubuntu (23.10), which is [necessary](https://answers.launchpad.net/ubuntu/+source/glibc/+question/708407) to *safely* upgrade to GLIBC_2.38.

Chat recommends using Docker to build an environment w/ newer GLIBC_2.38.  
Or should i just change OS from Pop -> Ubuntu 23.10.

Thoughts?

Reply:
ubuntu is a different operating system from pop

you'd need to look at the pop docs to see how to upgrade/install software on your machine

or just do the easy thing and run a more popular distro ;)

Reply:
I'll rip of the bandaid